metadata.json

```diff
@@ -56,7 +57,6 @@
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "9",
```

Comment:
I don't mind dropping Debian 9 (since it's EOL), but a separate PR is nice for the changelog. While you're at it: Ubuntu 18.04 is also EOL and can be dropped.

Reply:
done separately in https://github.com/theforeman/puppet-tftp/pull/142